cocos/serialization/instantiate.ts

```diff
@@ -28,7 +28,8 @@ import { getError, warn } from '@base/debug';
 import { cclegacy } from '@base/global';
 import { js } from '@base/utils';
 import { isDomNode } from '@pal/utils';
-import { CCObject, isCCObject, ValueType, jsbUtils, isCCClassOrFastDefined } from '../core';
+import { CCObject, isCCClassOrFastDefined, ValueType, isCCObject } from '@base/object';
```

Comment:
As `isCCClassOrFastDefined ` is something related with class, not object. It is better to implement in core.

Reply:
yes, it should be, but the implementation of CCObject rely on CCClass